openedx.yaml

```diff
@@ -1,13 +1,16 @@
 # This file describes this Open edX repo, as described in OEP-2:
 # http://open-edx-proposals.readthedocs.io/en/latest/oeps/oep-0002.html#specification
 
-owner: tuchfarber
-supporting_teams: edx/enterprise
+owner: schenedx
+supporting_teams:
+  - masters-devs
+  - edx/enterprise
 nick: desg
+tags:
+  - webservice
 oeps:
   oep-2: True
   oep-7: True
+  oep-10: True
```

Comment:
I think you want to delete this line, too, if it's not meant to be part of an Open edX release.

Reply:
...or maybe set it to `False`?